unlock-app/src/components/content/events-collection/CollectionCreationStatus.tsx

```diff
@@ -0,0 +1,54 @@
+import { Button } from '@unlock-protocol/ui'
+import { useRouter } from 'next/navigation'
+import { AnimationContent } from '~/components/interface/locks/Create/elements/CreateLockFormSummary'
+
+interface CollectionCreationStatusProps {
+  isCreating: boolean
+  success: boolean
+  createdSlug: string | null
+}
+
+export const CollectionCreationStatus = ({
+  isCreating,
+  success,
+  createdSlug,
+}: CollectionCreationStatusProps) => {
+  const router = useRouter()
+
+  if (isCreating) {
```

Comment:
That's probably a very short moment, isn't it?
I worry this may create a "flash" where users don't have time to read the stuff below.

Reply:
true! now replaced with a different approach!